ReactAndroid/src/main/java/com/facebook/react/views/scroll/ReactScrollViewHelper.java

```diff
@@ -96,4 +100,28 @@ public static int parseOverScrollMode(String jsOverScrollMode) {
       throw new JSApplicationIllegalArgumentException("wrong overScrollMode: " + jsOverScrollMode);
     }
   }
+
+  /**
+   * Helper method for animating to a ScrollView position with a given duration,
+   * instead of using "smoothScrollTo", which does not expose a duration argument.
+   */
+  public static ObjectAnimator animateScroll(final ViewGroup scrollView, int mDestX, int mDestY, int mDuration) {
```

Comment:
I have no idea if this Animator object is legit - @janicduplessis or @mdvacca?

Reply:
It has support from API 11:

https://developer.android.com/reference/android/animation/ObjectAnimator.html

FWIW we've been using this ScrollView (via overwriting the ScrollView package in MainApplication.Java) in production for the past 6 weeks, in production / release. We also use it in a fairly expensive SectionList which does all sorts of crazy optimizations and is a good stress test for performance.

Reply:
I would still like signoff from someone that knows our android codebase...@janicduplessis, @mdvacca, or @axe-fb?

Reply:
Sounds good.